### PR TITLE
Metrics widget page tests

### PR DESCRIPTION
**Pull request by davidverdu**
- Create e2e tests for the Metrics page
- Remove unneeded tasks from Makefile
- Create config for https://b-dev-cannonooda-02.rc.fas.harvard.edu/ server

image

image

### CODE REVIEW

**Comment by abujeda** [comment]
Could you add the same test for Sid, even if it is just a copy of the FASRC test.

This metrics page and link need to be in both profiles

**Comment by davidverdu** [comment]
@abujeda I included the tests for Sid profile. All the tests pass in dev environment

image

File: cypress/e2e/ondemand/sid/metrics.cy.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { NAVIGATION, loadHomepage, navigateToMetrics } from "../../../support/utils/navigation.js";
+import {changeProfile} from "../../../support/utils/profiles";
+
+describe('FASRC Dashboard - Metrics Widget and Page', () => {
```
Comment (abujeda) [inline cypress/e2e/ondemand/sid/metrics.cy.js:4]: This should be called `Sid Dashboard`

File: cypress/e2e/ondemand/sid/metrics.cy.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    changeProfile(cy.sid.profiles.sid.title)
+  })
+
+  beforeEach(() => {
+    //DEFAULT SIZE FOR THESE TESTS
+    cy.viewport(cy.sid.screen.largeWidth, cy.sid.screen.height)
+    loadHomepage()
+  })
+
+  describe(`${fasrcClusterProfile}: Should display Metrics page`, () => {
```
Comment (abujeda) [inline cypress/e2e/ondemand/sid/metrics.cy.js:19]: For Sid, we only need the name of the test, no need to display `${fasrcClusterProfile}: `
For consistency, you can take a look at other Sid test to compare

File: cypress/e2e/ondemand/fasrc/metrics.cy.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      cy.get('div.metrics-help').contains('h3','Metrics Help').should('be.visible')
+      cy.get('div.metrics-help div.introduction').should('be.visible')
+      cy.get('div.metrics-help div.description').should('be.visible') 
+      cy.get('div.metrics-help div.description').contains('h5','FairShare').should('be.visible') 
+      cy.get('div.metrics-help div.description').contains('h5','Completed Jobs by State').should('be.visible') 
+      cy.get('div.metrics-help div.description').contains('h5','Summary Job Stats').should('be.visible') 
+      cy.get('div.metrics-help div.description').contains('strong','Understanding Efficiency Metrics').should('be.visible')       
+    })
+
+    it('Right lateral section', () => {
```
Comment (abujeda) [inline cypress/e2e/ondemand/fasrc/metrics.cy.js:33]: Same comment as Sid

**Comment by abujeda** [comment]
Once the different titles of the tests are fixed, this is ready to go!

File: cypress/e2e/ondemand/fasrc/metrics.cy.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import { NAVIGATION, loadHomepage, navigateToMetrics } from "../../../support/utils/navigation.js";
+import {changeProfile} from "../../../support/utils/profiles";
+
+describe('FASRC Dashboard - Metrics Widget and Page', () => {
+  const fasrcClusterProfile = Cypress.env('fasrc_cluster_profile')
+  Cypress.config('baseUrl', NAVIGATION.baseUrl);
+  const cluster = Cypress.env('cluster')
+  const clusterName = cluster == 'cannon' ? 'Cannon' : cluster == 'fasse' ? 'FASSE' : 'Example'
```
Comment (abujeda) [inline cypress/e2e/ondemand/fasrc/metrics.cy.js:8]: ClusterName should be configured in the environment specific properties file.
Similar to the welcome message:
https://github.com/hmdc/ondemand_testing/blob/main/ondemand/cypress.env.json.prod-cannon#L8

File: cypress/plugins/index.js [hunk 5]
```diff
@@ -72,6 +73,7 @@ module.exports = (on, config) => {
 
   config.baseUrl = app_url
   config.env['baseUrl'] = app_url
+  config.env['cluster'] = ood_environment.split('-')[1].split('.')[0]
```
Comment (abujeda) [inline cypress/plugins/index.js:76]: As per comment above, this is not required

**Pull request merged**